src/context.rs

```diff
@@ -574,6 +556,67 @@ pub enum Shell {
     Unknown,
 }
 
+fn default_width() -> usize {
+    terminal_size().map_or(80, |(w, _)| w.0 as usize)
+}
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum Target {
+    Main,
+    Right,
+    Continuation,
+}
+
+/// Properties as passed on from the shell as arguments
+#[derive(Parser, Debug)]
+
```

Comment:
nitpick: Looks like a lingering newline was left here between the attribute and the definition 😄 
```suggestion
```